code/modules/mining/ores_coins.dm

```diff
@@ -40,6 +40,17 @@
 	points = 1
 	refined_type = /obj/item/stack/sheet/glass
 
+/obj/item/weapon/ore/glass/throw_impact(A)  //to make eye blurring possible with throwing sand.
+	if(istype(A,/mob/living/carbon/human))  //we can add a conditional later to check if the "thrower" is targeting the eyes, but I felt it was uneeded.
+		var/mob/living/carbon/human/T = A
+		if(T.check_part_covered("eyes"))
+			T.eye_blurry = 100
+			A << "<span class='warning'>The sand hits your eyes blinding you!</span>"
+		else
+			A << "<span class='warning'>The sand hits your mask!</span>"
+			qdel(src) //dissipates?
```

Comment:
If we're going to consume the sand, let's make it happen regardless ( remove a tab ).